client/src/components/App.css

```diff
@@ -0,0 +1,4 @@
+.container {
+    width: 100%!important;
+    max-width: 100%;
+}
```

Comment:
I think it's overriding materialize's CSS. Otherwise the header's width won't reach either ends of the window. I'd like to look into ways to make the container affect everything other than the header.

Reply:
If we're not using Materialize's container why use it at all is my point